src/controllers/view/publicController.js

```diff
@@ -0,0 +1,35 @@
+const { Comments } = require("../../models");
+const Blogs = require("../../models/Blogs");
+
+const renderLogin = (req, res) => {
```

Comment:
Consider using a try catch block here and in the `renderSignUp` function that will throw an error if the page cannot be rendered.